PR: https://github.com/odoo/odoo/pull/

From: e38e2515882dacc73ab7d87fa76da3c5a395c6a6
From: Mahendra barad
Date: 2017-09-06 13:21:25

Structural Changes: 7
Total Changes: 16

[IMP] website: Clean and simplify res_config view

Purpose
=======

- Settings are too complex: too long + too many options that shouldn't be suggested with checkboxes

e.g. twitter roller is now suggested as a new snippet to install from website editor
google maps, slides, forum, etc. should show up in the apps store

- When saving the page I expect to stay in the settings -> no redirection to homepage anymore!

Specification
=============

See task 33620

================================= pseudo patch: =================================

--- a/addons/mass_mailing/models/__init__.py
+++ b/addons/mass_mailing/models/__init__.py
@@ -8,3 +8,4 @@ from . import mail_thread
 from . import res_config_settings
 from . import mass_mailing_report
 from . import link_tracker
+from . import res_company

--- a/None
+++ b/addons/mass_mailing/models/res_company.py
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    # NB: Also defined in website
+    social_twitter = fields.Char('Twitter Account')
+    social_facebook = fields.Char('Facebook Account')
+    social_github = fields.Char('GitHub Account')
+    social_linkedin = fields.Char('LinkedIn Account')
+    social_youtube = fields.Char('Youtube Account')
+    social_googleplus = fields.Char('Google+ Account')
